src/chart/pie/PieSeries.ts

```diff
@@ -112,6 +112,7 @@ export interface PieSeriesOption extends
 
     clockwise?: boolean
     startAngle?: number
+    angleRange?: number
```

Comment:
Although I do think `angleRange` may be easier to be understood than `endAngle`, we still need to comply with currect option of [gauge.endAngle](https://echarts.apache.org/zh/option.html#series-gauge.endAngle).

Reply:
Yes, at first I was going to use `endAngle`, but since `startAngle` has been around for a long time, using `endAngle` would probably cause breaking change. if `startAngle` is set to a non-default value and `endAngle` is still the default, it's no longer a complete pie chart. So I feel that endAngle may not be a particularly good way to go. Of course naming consistency is a great thing.

Reply:
Could we make the default value of `endAngle` to be `'auto'` to avoid this?

Reply:
Sorry, I ignored this option, I use `endAngle` instead of `angleRange`, and default value is `'auto'`.